### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by dmosites** [airswap/airswap-web#131] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gpxl-dev** [comment]
Hey! 👋 

I'm Ed, or "greypixel" on the AirSwap discord. I live in the UK, a couple of miles down the road from where they film "Downton Abbey"! By day I run a small tech business, wearing many different hats, but my academic background is in Engineering so I like to keep very involved with the technical side wherever possible.

I also have a 1 year old daughter and have recently found that "Free time" is a bit of a foreign concept, but since March this year whenever I've had the opportunity I've been contributing to the AirSwap community. Most recently I've been focussing on developing the new web app, working with the other awesome contributors, but I've also been working on a few other smaller things like the "big swap bot" on discord, and have been trying to play a part in governance too.

Really enjoying being a part of it and would urge anyone who is thinking about getting involved to do so!
